src/azure-firewall/azext_firewall/custom.py

```diff
@@ -122,6 +151,29 @@ def update_azure_firewall(cmd, instance, tags=None, zones=None, private_ranges=N
         instance.additional_properties['Network.DNS.Servers'] = ','.join(dns_servers or '')
     if threat_intel_mode is not None:
         instance.threat_intel_mode = threat_intel_mode
+
+    if instance.hub_ip_addresses is None and hub_public_ip_addresses is not None:
```

Comment:
To make users easy to understand, can we tell users that they can only provide one of count (add) and ip_addresses (delete)? provide both help doc and verification logic.